### PR TITLE
chore: fix pulsar addon typo

### PR DESCRIPTION
**Pull request by Y-Rookie**
(cherry picked from commit 002fd9598d2cdc7e365c42bfe72ef13003bd4419)

### CODE REVIEW

File: addons/pulsar/scripts/init-broker.sh [hunk 1]
```diff
@@ -85,11 +85,6 @@ init_broker() {
   fi
 
   initialize_cluster_metadata "$clusterName" "$zookeeperServers" "$webServiceUrl" "$brokerServiceUrl"
-  quit_script
-}
-
-quit_script() {
```
Comment (zjx20) [inline addons/pulsar/scripts/init-broker.sh:91]: This function still appears in the file (a few lines above).

Comment (Y-Rookie) [reply addons/pulsar/scripts/init-broker.sh:91]: > This function still appears in the file (a few lines above).

fixed done

File: addons/pulsar/scripts/init-broker.sh [hunk 2]
```diff
@@ -77,11 +77,11 @@ init_broker() {
   local idx=${POD_NAME##*-}
   if [ $idx -ne 0 ]; then
     wait_for_cluster_metadata "$zookeeperServers" "$clusterName"
-    echo "Cluster already initialized" && quit_script
```
Comment (zjx20) [inline addons/pulsar/scripts/init-broker.sh:80]: Change to `&& exit 0` to keep the semantic.

Comment (Y-Rookie) [reply addons/pulsar/scripts/init-broker.sh:80]: > Change to `&& exit 0` to keep the semantic.

fixed done

**Review by zjx20 — APPROVED** [review]

**Review by wangyelei — APPROVED** [review]

**Pull request merged**